Wikipedia/Code/OnThisDayViewController.swift

```diff
@@ -88,6 +115,34 @@ class OnThisDayViewController: ColumnarCollectionViewController, DetailPresentin
             FeedFunnel.shared.logFeedCardClosed(for: feedFunnelContext, maxViewed: maxViewed)
         }
     }
+
+    private func addUnderBarHeader() {
+        let headerView = ThreeLineHeaderView()
+        headerView.apply(theme: theme)
+
+        let language = events.first?.language
+        let locale = NSLocale.wmf_locale(for: language)
+        let semanticContentAttribute = MWLanguageInfo.semanticContentAttribute(forWMFLanguage: language)
+
+        headerView.topSmallLine.semanticContentAttribute = semanticContentAttribute
+        headerView.middleLargeLine.semanticContentAttribute = semanticContentAttribute
+        headerView.bottomSmallLine.semanticContentAttribute = semanticContentAttribute
+
+        headerView.topSmallLine.text = CommonStrings.onThisDayTitle.uppercased()
+
+        let eventCountText = String(format: WMFLocalizedString("on-this-day-detail-header-title", language: language, value:"{{PLURAL:%1$d|%1$d historical event|%1$d historical events}}", comment:"Title for 'On this day' detail view - %1$d is replaced with the number of historical events which occurred on the given day"), locale: locale, events.count).uppercased(with: locale)
```

Comment:
I suggest moving this WMFLocalizedString (and the one in OnThisDayViewControllerHeader) to CommonStrings.swift.